tests/smoke-tests/src/test/java/org/apache/activemq/artemis/tests/smoke/console/QueuesTest.java

```diff
@@ -60,6 +62,89 @@ public void testDefaultQueues() throws Exception {
       assertEquals(0, queuesPage.getMessagesCount("ExpiryQueue"));
    }
 
+   @Test
+   public void testConnectionsTab() {
```

Comment:
Nice tests but they don't seem related to queues, could you move them to a new test class, i.e. `TabsTest` or any other names?